### ISSUE TITLE
Focus movement with Attached file pill should be improved

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#234194]
1. Open chat view, notice current file is attached and rendered as a pill
2. Press <kbd>Tab</kbd> -> focus is nicely moved to the pill
3. Press <kbd>Tab</kbd> focus moves to the eye
4. Press space on the eye. Focus gets lost 🐛  Instead focus should stay on the pill
5. Press <kbd>esc</kbd> when focus is on the pill. It should move focus back to chat like find widget does 🐛 

In addition we need to make it easier for users to remove / disable items. This can follow the breakpoints pattern.
* When focus is on the pill, <kbd>cmd</kbd>+<kbd>backspace</kbd> on macOS and <kbd>delete</kbd> on Win and LInux

Fixing this should make it easier for a keyboard centric user to disable / enable attached files. And move focus back to input.

fyi @ulugbekna @meganrogge
